news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+<a id="conclusion"></a>
+
+---
+
+And that concludes the last issue of the Mappers' Report in the year 2020! Quite a lot to read again, but we are happy should you have survived until here.
+
+To the end of the year, the lead of the Mappers' Report [Feerum](https://osu.ppy.sh/users/4815717), would like to drop a few words: 
+
+The Mappers' Report started as a project together with the Beatmap Nominators out of a whim after a NAT Meeting. First we NAT just wanted to use this as a plattform to reach more participants for the meetings and inform these who couldn't participate about what exactly was discussed. This worked by the way! 
+But over the time i have gotten more ideas for the whole project and it became huge! I added more cathegories because i wanted to bring as much informations about the, in my eyes, most important aspect in osu! to as many people as possible. 
+
+This whole project would not have been that successful as it is now without the help of so many people. I wish i could thank everyone who helped in a proper way, but sadly i can't name all because the ending here would become as large as the whole news post. Be sure that i am thankful for really everyone who helped me with it in the past, either by writing sections or picking the mapper for the Mappers' Showcase. But i also want to mention a few special people here: 
+
+First of all a lot of thanks to [Noffy](https://osu.ppy.sh/users/1541323) who supported it from the very beginning and helped a lot with writing. I could never make this news post such a big success without you. So thousand thanks that you stayed all the time and helped me with the Mappers' Report.
+
+Next i would like to thank the whole osu!wiki team which is helping me by proof reading the whole news post once the pull request has been opened and by helping me to get rid of so many grammar mistakes i did and making the whole news post overall more comfortable to read. I am not a native English speaker but i still work on improving it! So thank you all for the help and i hope i can count on you in the future too.
+
+Last but not least i would like to thank all readers of the Mappers' Report. A news post is nothing without the people who want to read it. Every time we have gotten late with it, i got a lot of private messages by people asking when the next issue will be released. This time is no exception. I am sorry that i make you wait. This project has become more important to me than i could have imagine and that's only thank to all your positive feedback and support. Thank you so much and i hope you will continue to stay with us in the next year as well!
```

Comment:
"Every time we have been late with it, I have received a lot [...]"

I **made** you wait

imagine**d** and that's only thank**s**